complex/cargo/list/src/items.rs

```diff
@@ -15,17 +15,30 @@ use std::os::raw::{
 };
 use std::ptr;
 
+pub use edn::{
+    DateTime,
+    Utc,
+};
+use mentat_core::{
```

Comment:
Are these re-exported from `mentat`, the top-level crate? If not we should probably do so.